arm-network/2016-09-01/swagger/network.json

```diff
@@ -7900,6 +7898,43 @@
       },
       "description": "VpnClientParameters"
     },
+    "TunnelConnectionHealth": {
+      "properties": {
```

Comment:
Looks like all the properties of this model are readOnly properties. 
Since Network service uses the same model as input and output for PUT operation hence specifying the readOnly (ness) is very important.
When you say that a particular property is `"readOnly": true` in swagger, it means that the property will not be sent on the wire while sending the request. It can only be populated by the server in the response.